recipes-connectivity/awsclient/awsclient_1.3.1.bb

```diff
@@ -0,0 +1,35 @@
+SUMMARY = "The AWS IoT client establishes a connection to AWS IoT, executes jobs and sends status information."
+
+LICENSE = "CLOSED"
+
+SRC_URI = "\
+    https://github.com/osb-cc-esec/meta-esec-awsclient/releases/download/v${PV}/awsclient_${PV}_aarch64 \
```

Comment:
I think we need some kind of automatic parsing for the architecture as well. Consider using `TARGET_ARCH`, `TUNE_ARCH` or something similar instead of hardcoding `aarch64`.